test_plans/bfloat16.asciidoc

```diff
@@ -0,0 +1,118 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for bfloat16
+
+This is a test plan for the APIs described in
+https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc[sycl_ext_oneapi_bfloat16]
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+=== Type coverage
+
+All of the tests described below are performed using each of the following `typename T`:
+
+* `char`
+* `short`
+* `int`
+* `long long`
+* `std::size_t`
+* `bool`
+* `float`
+
+In addition, if the device has aspect `aspect::fp64`:
+
+* `double`
+
+In addition, if the device has `aspect::fp16`:
+
+* `sycl::half`
+
+Note that `T` must be convertible to `float`
+
+== Tests
+
+=== Constructors
+
+`bfloat16()` +
+`bfloat16(const bfloat16 &)`
+
+Check that:
+
+* `std::is_default_constructible_v<bfloat16> == true`.
+* `std::is_copy_constructible_v<bfloat16> == true`.
+
+`bfloat16(const float &a)` +
+`bfloat16(const sycl::half &a)`
```

Comment:
How are you testing that it is possible to construct a `bfloat16` from `float` or `sycl::half`?  The description below doesn't explain this.  There should also be tests to verify the assignment operators taking `float` and `sycl::half`.

Reply:
I will clarify this section